### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by rcnijzink**
It's not really a bug, but I changed the last soil layer to align with the surface

### CODE REVIEW

File: VOM_Fortran/VOM-code/transpmodel.f90 [hunk 1]
```diff
@@ -638,6 +654,59 @@ subroutine vom_get_soilprofile ()
       return
       end subroutine vom_get_soilprofile
 
+!+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+!+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+!+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+!*-----READING TIMESERIES OF VEGETATION COVER---------------------------
```
Comment (schymans) [inline VOM_Fortran/VOM-code/transpmodel.f90]: Should be: READING TIMESERIES OF SEASONAL VEGETATION COVER

**Review by schymans — CHANGES_REQUESTED** [review]
I only see the change to allow reading pcg from a file. 
This is a great addition! Could you also mention the changes in the documentation and state clearly that the file with seasonal vegetation cover should refer only to pcg, i.e. cover of seasonal vegetation, not perennial. If a remote-sensing derived data set is used, need to separate the perennial fraction from the seasonal fraction and provide the perennial in the input parameter file, while the seasonal will be read in from the time series file.

What was the problem about aligning the top soil layer with the surface?

**Comment by rcnijzink** [comment]
ooh oops... this still needed testing and wasn't supposed to be in this pull request... it was just about aligning s_delz with the surface. Maybe just accept this change for now:

's_delz(s_maxlayer) = i_cz - (s_maxlayer - 1) * i_delz'

I will make a separate pull-request for reading in the vegation timeseries.

**Comment by schymans** [comment]
Could you create a separate pull request for that line? Sorry, but I don't know if there is a way to merge only part of a PR.

**Comment by rcnijzink** [comment]
Probably not the most elegant way of solving this, but I removed the code that you don't need for this bugfix and commited again (it's in another branch anyway). So if you would accept the pull request now it only includes the bugfix (based on the last commit)

**Pull request merged**